CHANGELOG.next.asciidoc

```diff
@@ -206,6 +206,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Fixed data types for various hosts fields in `mongodb/replstatus` metricset {pull}10307[10307]
 - Added function to close sql database connection. {pull}10355[10355]
 - Fix issue with `elasticsearch/node_stats` metricset (x-pack) not indexing `source_node` field. {pull}10639[10639]
+- Fix flaky test on Agent Metricset on Consul module {pull}10766[10766]
```

Comment:
Flaky test fixes are not needed in the changelog as this has no impact on users.

Reply:
Removed, sorry!